### PR TITLE
WIP - Cue template to create kyverno clusterpolicy

### PR DESCRIPTION
**Pull request by pxp928**
Work in progress. Do not merge.

### CODE REVIEW

File: platform/30-kyverno-setup.sh [hunk 1]
```diff
@@ -55,5 +74,5 @@ kubectl patch \
   --type json --patch-file "${GIT_ROOT}"/platform/components/kyverno/patch_container_args.json
 
 echo -e "${C_GREEN}Creating verify-image admission control policy...${C_RESET_ALL}"
-cue export "$KYVERNO_RESOURCE_DIR"/admission-control-verify-image-resources.cue -e template -t key="$USERPUBKEY" --out yaml > "$KYVERNO_RESOURCE_DIR"/admission-control-verify-image-resources.yaml
-kubectl apply -f "$KYVERNO_RESOURCE_DIR"
+cue eval ./... -t repo="$REPO" -t key="$USERPUBKEY" --out yaml > $KYVERNO_RESOURCE_DIR/released/admission-control.yaml
```
Comment (bradbeck) [inline platform/30-kyverno-setup.sh:77]:
```suggestion
pushd "$KYVERNO_RESOURCE_DIR"
cue eval ./... -t repo="$REPO" -t key="$USERPUBKEY" --out yaml > $KYVERNO_RESOURCE_DIR/released/admission-control.yaml
popd
```

File: platform/30-kyverno-setup.sh [hunk 2]
```diff
@@ -6,6 +6,25 @@ MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhyQCx0E9wQWSFI9ULGwy3BuRklnt
...
+C_RED='\033[31m'
+C_RESET_ALL='\033[0m'
+
+# Wait until pods are ready.
+# $1: namespace, $2: app label
+wait_for_pods () {
+  echo -e "${C_YELLOW}Waiting: $2 pods in $1...${C_RESET_ALL}"
+  kubectl wait --timeout=5m --for=condition=ready pods -l app=$2 -n $1
+}
+
```
Comment (bradbeck) [inline platform/30-kyverno-setup.sh:27]: I think this part is duplicated below.

Comment (pxp928) [reply platform/30-kyverno-setup.sh:27]: Yea not sure what happened with the upstream merger. I will need to squash commits so I'll get it all sorted.

File: platform/30-kyverno-setup.sh (hunk 1 above)
Comment (pxp928) [reply platform/30-kyverno-setup.sh:77]: Yup I needed to fix that. I was running all by hand.

**Comment by pxp928** [comment]
Closing PR to update.

**Pull request closed without merging**